package.json

```diff
@@ -26,7 +26,7 @@
     "build-viewer": "node ./build/build-viewer.js",
     "reset-link": "(yarn unlink || true) && yarn link && yarn link lighthouse",
     "c8": "bash lighthouse-core/scripts/c8.sh",
-    "clean": "rimraf dist proto/scripts/*.json proto/scripts/*_pb2.* proto/scripts/*_pb.* proto/scripts/__pycache__ proto/scripts/*.pyc *.report.html *.report.dom.html *.report.json *.devtoolslog.json *.trace.json lighthouse-core/lib/i18n/locales/*.ctc.json || true",
+    "clean": "rm -r dist proto/scripts/*.json proto/scripts/*_pb2.* proto/scripts/*_pb.* proto/scripts/__pycache__ proto/scripts/*.pyc *.report.html *.report.dom.html *.report.json *.devtoolslog.json *.trace.json lighthouse-core/lib/i18n/locales/*.ctc.json || true",
```

Comment:
breaks for windows, doesn't it?

Reply:
dont really care if it does. seems reasonable to support only bash-in-windows and not the windows command line.

Reply:
> dont really care if it does. seems reasonable to support only bash-in-windows and not the windows command line.

Yeah, I also don't think it matters. I honestly never run this because it also deletes `*.report.html *.report.json *.devtoolslog.json *.trace.json` and I usually have at least one of those lying around I don't want deleted right now, but I'm sure someone out there likes a tidy checkout.